kolibri/plugins/coach/assets/src/views/reports/align-mixin.js

```diff
@@ -0,0 +1,9 @@
+import Vue from 'kolibri.lib.vue';
+
+export default {
+  computed: {
+    align() {
+      return Vue.bidi === 'rtl' ? 'right' : 'left';
+    },
```

Comment:
Changed to `alignStart` to be more semantic - did not add an as yet unused `alignEnd` in the interests of not overgeneralizing too early.